### PR TITLE
Split cannon benchmarks and tests

### PR DESCRIPTION
**Pull request by pcapriotti**
Cannon benchmarks were running as part of the test suite. This creates a separate component for cannon benchmarks, so that they are not run alonside the other unit tests.

## Checklist

 - [x] The **PR Title** explains the impact of the change.
 - [x] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
 - [x] **changelog.d** contains the following bits of information ([details](https://github.com/wireapp/wire-server/blob/develop/docs/developer/changelog.md)):
   - [x] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.

### CODE REVIEW

File: services/cannon/bench/Main.hs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import Bench
+import qualified Cannon.API
+import Data.Metrics.Test (pathsConsistencyCheck)
+import Data.Metrics.WaiRoute (treeToPaths)
+import Imports
+import Network.Wai.Utilities.Server (compile)
+import qualified Test.Cannon.Dict as D
+import Test.Tasty
+import Test.Tasty.HUnit
```
Comment (supersven) [inline services/cannon/bench/Main.hs:28]: Do you need all these imports? The only imported symbol I see is `benchmark`... :thinking:

Comment (supersven) [reply services/cannon/bench/Main.hs:28]: I think `import Bench` should be sufficient.

**Review by supersven — CHANGES_REQUESTED** [review]

File: services/cannon/bench/Main.hs (hunk 1 above)
Comment (pcapriotti) [reply services/cannon/bench/Main.hs]: Yes, fixed in aed403d69ca497f01d590ac4efdb036cf0a418d3.

**Comment by supersven** [comment]
Looks good to me. :+1: 

Let's wait for the CI results. When they're positive, I'll approve.

**Review by supersven — APPROVED** [review]

**Pull request merged**